### PR TITLE
Test commit

### PR DESCRIPTION
**Pull request by benhoyt**
This is a test comment to test stuff.

### CODE REVIEW

File: soft404.py [hunk 1]
```diff
@@ -42,7 +40,7 @@ def almost_identical(html1, html2, minratio=IDENTICAL_RATIO):
     >>> almost_identical(h1, h2)
     False
     """
-    seq1 = html1.split()
+    seq1 = html1.split()  # this is a test comment
```
Comment (benhoyt-test) [inline soft404.py]: I'm not sure this comment is necessary

File: soft404.py [hunk 2]
```diff
@@ -7,9 +7,7 @@
 
 """
 
-import difflib
-import httplib
-import random
+import difflib, httplib, random
```
Comment (benhoyt-test) [inline soft404.py:10]: Why the reformatting change?

File: soft404.py [hunk 3]
```diff
@@ -60,6 +58,12 @@ def random_letters(n):
     letter_list = [random.choice(string.ascii_lowercase) for i in range(n)]
     return ''.join(letter_list)
 
+
+def add(a, b):
+    """Add two numbers together."""
```
Comment (benhoyt-test) [inline soft404.py:63]: Excellent doc string!

**Review by benhoyt-test — COMMENTED** [review]
This is great work (for at test).

File: soft404.py (hunk 2 above)
Comment (benhoyt) [reply soft404.py:10]: Good question. I thought long and hard about this one, and came to the conclusion that it was necessary.

File: soft404.py (hunk 3 above)
Comment (benhoyt) [reply soft404.py:63]: Ack

File: soft404.py (hunk 1 above)
Comment (benhoyt) [reply soft404.py]: Fixed

**Pull request closed without merging**